src/fsharp/service/ItemKey.fsi

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.SourceCodeServices
+
+open System
+open FSharp.Compiler.Range
+open FSharp.Compiler.NameResolution
+
+[<Sealed>]
+type internal ItemKeyStore =
```

Comment:
Probably good to have `///` comments on the types declared in this file for future readers of the source code

Reply:
Yes. I'll put a comment here.